### PR TITLE
[Whisper fine-tuning sprint] README updates + organisation

### PR DESCRIPTION
**Pull request by Vaibhavs10**
**Motivation**: One-stop resource for all the resources and instructions for the sprint.

**Intended audience**: Participants for the Whisper community event and future developers/ hobbyist chancing upon fine-tuning guides for the event.

**Estimated date for release**: Monday, 21st November 2022

### CODE REVIEW

File: whisper-fine-tuning-sprint/README.md [hunk 1]
```diff
@@ -0,0 +1,617 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:6]: Fingers crossed 🤞

File: whisper-fine-tuning-sprint/README.md [hunk 2]
```diff
@@ -0,0 +1,617 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:5]: Maybe we can be a bit more specific here? i.e. relate to Whisper and achieving SoTA 70+ langs?

File: whisper-fine-tuning-sprint/README.md [hunk 3]
```diff
@@ -0,0 +1,635 @@
...
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
+This document summarizes all the relevant information required for the speech community event 📋.
+
+To sign-up, please see [this forum post](https://LINK_ACTUAL_FORUM) 🤗. Please make sure to:
+- Read it in detail
+- Fill the [google form](https://forms.gle/F2bpouvhDpKKisM39)
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:11]:
```suggestion
- Fill the [Google form](https://forms.gle/F2bpouvhDpKKisM39)
```

File: whisper-fine-tuning-sprint/README.md [hunk 4]
```diff
@@ -0,0 +1,635 @@
...
+- [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model)
+- [How to fine-tune with OVH could](#how-to-finetune-with-ovh-cloud)
+- [How to combine n-gram language models with acoustic model](#how-to-combine-n-gram-with-acoustic-model)
+- [Evaluation](#evaluation)
+- [Prizes](#prizes)
+- [Communication and Problems](#communication-and-problems)
+- [Talks](#talks)
+- [General Tips & Tricks](#general-tips-and-tricks)
+
+## TLDR
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:31]: Consistent with the Table of Contents
```suggestion
## TLDR;
```

File: whisper-fine-tuning-sprint/README.md [hunk 5]
```diff
@@ -0,0 +1,635 @@
...
+
+[comment]: # TODO: VB - create a post on the forum and link it above. Update the URL to cloud provider when decided.
+
+## Table of Contents
+
+- [TLDR;](#tldr)
+- [Important dates](#important-dates)
+- [How to install pytorch, transformers, datasets](#how-to-install-relevant-libraries)
+- [Data and Preprocessing](#data-and-preprocessing)
+- [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model)
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:22]: Would collapse this down to 'How to fine-tune a Whisper model'

File: whisper-fine-tuning-sprint/README.md [hunk 6]
```diff
@@ -0,0 +1,635 @@
...
+- [How to combine n-gram language models with acoustic model](#how-to-combine-n-gram-with-acoustic-model)
+- [Evaluation](#evaluation)
+- [Prizes](#prizes)
+- [Communication and Problems](#communication-and-problems)
+- [Talks](#talks)
+- [General Tips & Tricks](#general-tips-and-tricks)
+
+## TLDR
+
+Participants are encouraged to leverage pre-trained speech recognition checkpoints,
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:33]: This is a bit different to the robust speech event where the theme was a more broad (take any ASR model and fine-tune).

I would be more direct here in explaining how the event is going to work. I would state that participants are going use the pre-trained Whisper checkpoints (link to checkpoints) and fine-tune... 

-> want to make very clear how the event is set-up

File: whisper-fine-tuning-sprint/README.md [hunk 7]
```diff
@@ -0,0 +1,635 @@
...
+preferably [openai/whisper-large](https://huggingface.co/openai/whisper-large),
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:34]: Maybe copy and past the table from the blog with checkpoints and params? https://huggingface.co/blog/fine-tune-whisper#introduction

Also probably medium is going to be the largest we can feasibly train on an A100 -> this is fine as medium performs very similarly to large on most languages

We can train large but it'll be very slow (high number of gradient accumulation steps) -> I think we're better off encouraging the smaller checkpoints (small and medium) for more training steps

File: whisper-fine-tuning-sprint/README.md [hunk 8]
```diff
@@ -0,0 +1,635 @@
...
+Participants are encouraged to leverage pre-trained speech recognition checkpoints,
+preferably [openai/whisper-large](https://huggingface.co/openai/whisper-large), 
+to train a speech recognition system in a language of their choice.
+
+Speech recognition systems should be trained using **PyTorch**, **🤗 Transformers**, and, **🤗 Datasets**.
+For more information on how to install the above libraries, please read through 
+[How to install pytorch, transformers, datasets](#how-to-install-relevant-libraries).
+
+Participants can make use of whatever data they think is useful to build a 
+speech recognition system for **real-world** audio data -
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:42]: I think the notion of 'real-world' audio data is pretty specific to the robust speech challenge! Ours is going to be more similar in theme to https://github.com/huggingface/transformers/blob/main/examples/research_projects/wav2vec2/FINE_TUNE_XLSR_WAV2VEC2.md#fine-tuning-week-of-xlsr-wav2vec2-on-60-languages-
i.e. more focussed on common voice

File: whisper-fine-tuning-sprint/README.md [hunk 9]
```diff
@@ -0,0 +1,635 @@
...
+
+Participants can make use of whatever data they think is useful to build a 
+speech recognition system for **real-world** audio data - 
+**except** the Common Voice `"test"` split of their chosen language.
+The section [Data and preprocessing](#data-and-preprocessing) explains 
+in more detail what audio data can be used, how to find suitable audio data, and 
+how the audio data can be processed.
+
+For training, it is recommended to use the [official training script](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py) or a modification thereof. A step-by-step guide on how to fine-tune 
+an acoustic model for a speech recognition system can be found under [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model).
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:49]: Swap all of the 'acoustic' for 'whisper'

File: whisper-fine-tuning-sprint/README.md [hunk 10]
```diff
@@ -0,0 +1,635 @@
...
+If possible it is encouraged to fine-tune the acoustic models on local GPU machines, but 
+if those are not available, the CLOUD_PROVIDER could team kindly provides a limited 
+number of GPUs for the event. Simply fill out [this google form](https://forms.gle/F2bpouvhDpKKisM39) to get access to a GPU.
+[comment]: # For more information on how to train an acoustic model on one of OVH's GPU - see [How to fine-tune a speech recognition model with OVHcould](#how-to-fine-tune-with-ovh-cloud).
+
+During the event, the speech recognition system will be evaluated on both the Common Voice `"test"` split 
+of the participants' chosen language as well as the *real-world* `"dev"` data provided by
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:56]: Real-world dev was something specific to robust speech. I don't think this was discussed for Whisper?

File: whisper-fine-tuning-sprint/README.md [hunk 11]
```diff
@@ -0,0 +1,635 @@
...
+[comment]: # VB - Create an infographic for the timeline.
+
+## Data and preprocessing
+
+In this section, we will quickly go over how to find suitable training data and 
+how to preprocess it.
+
+To begin with, **all data except Common Voice's `"test"` data can be used as training data.**
+The exception includes all Common Voice versions as the test data split of later Common Voice versions often
+overlaps with the one of previous versions, *e.g.* the test data of Common Voice 7 in English is
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:96]: I would compare Common Voice 11 with Common Voice 10 rather than 7 with 6

File: whisper-fine-tuning-sprint/README.md [hunk 12]
```diff
@@ -0,0 +1,635 @@
...
+```
+
+However, we strongly encourage participants to make use of Common Voice's other splits, *e.g.* `"train"` and `"validation"`.
+For most languages, the Common Voice dataset offers already a decent amount of training data. It is usually 
+always advantageous to collect additional data. To do so, the participants are in a first step encouraged to search the
+Hugging Face Hub for additional audio data, for example by selecting the category 
+["speech-processing"](https://huggingface.co/datasets?task_categories=task_categories:speech-processing&sort=downloads).
+All datasets that are available on the Hub can be downloaded via the 🤗 Datasets library in the same way Common Voice is downloaded.
+If one wants to combine multiple datasets for training, it might make sense to take a look at 
+the [`interleave_datasets`](https://huggingface.co/docs/datasets/package_reference/main_classes.html?highlight=interleave#datasets.interleave_datasets) function.
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:116]: We can also try and provide an example code snippet here!

File: whisper-fine-tuning-sprint/README.md [hunk 13]
```diff
@@ -0,0 +1,635 @@
...
+![timeline](https://github.com/patrickvonplaten/scientific_images/raw/master/Robush%20Speech%20Challenge.png)
+
+[comment]: # VB - Create an infographic for the timeline.
+
+## Data and preprocessing
+
+In this section, we will quickly go over how to find suitable training data and 
+how to preprocess it.
+
+To begin with, **all data except Common Voice's `"test"` data can be used as training data.**
```
Comment (sanchit-gandhi) [inline whisper-fine-tuning-sprint/README.md:94]: We could also add a hyperlink to Common Voice on the Hub

**Review by sanchit-gandhi — COMMENTED** [review]
Partial review just to give preliminary pointers. Will have a full read through in the coming days!

File: whisper-fine-tuning-sprint/README.md [hunk 14]
```diff
@@ -0,0 +1,635 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:3]:
```suggestion
Welcome to the Whisper fine-tuning sprint 🎙️!
```

File: whisper-fine-tuning-sprint/README.md (hunk 1 above)
Comment (osanseviero) [reply whisper-fine-tuning-sprint/README.md:6]: To unblock this PR, maybe just say this  

```suggestion
Free compute resources will be provided for those that cannot train models with their compute resources.  🚀.
```
(or something along those lines)

File: whisper-fine-tuning-sprint/README.md [hunk 15]
```diff
@@ -0,0 +1,635 @@
...
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
+This document summarizes all the relevant information required for the speech community event 📋.
+
+To sign-up, please see [this forum post](https://LINK_ACTUAL_FORUM) 🤗. Please make sure to:
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:9]: I don't think the forum post should be required for people here. Going through this document should be everything they need, otherwise, we'll lose people jumping from one link to another. This page has everything important I think

File: whisper-fine-tuning-sprint/README.md [hunk 16]
```diff
@@ -0,0 +1,635 @@
...
+- Read it in detail
+- Fill the [google form](https://forms.gle/F2bpouvhDpKKisM39)
+- Join the [Hugging Face Discord server](https://hf.co/join/discord) and make sure you have access to the #events channel.
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:12]: How? They need to go to the role-assignments channel and accept audio

File: whisper-fine-tuning-sprint/README.md [hunk 17]
```diff
@@ -0,0 +1,635 @@
...
+
+[comment]: # TODO: VB - create a post on the forum and link it above. Update the URL to cloud provider when decided.
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:14]: To update before merge

(as mentioned above, I don't think the forum post is a requisite for this)

File: whisper-fine-tuning-sprint/README.md [hunk 18]
```diff
@@ -0,0 +1,635 @@
...
+
+## Table of Contents
+
+- [TLDR;](#tldr)
+- [Important dates](#important-dates)
+- [How to install pytorch, transformers, datasets](#how-to-install-relevant-libraries)
+- [Data and Preprocessing](#data-and-preprocessing)
+- [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model)
+- [How to fine-tune with OVH could](#how-to-finetune-with-ovh-cloud)
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:23]: Maybe let's remove this section for now, it could say Compute Resources with a big TODO

File: whisper-fine-tuning-sprint/README.md (hunk 4 above)
Comment (osanseviero) [reply whisper-fine-tuning-sprint/README.md:31]: Then the link will also need to be updated just in case!

Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:31]: This is a very large TL;DR section, maybe let's do an actual TL;DR with a 3-4 sentences description of the event and then this section could be named Intro or Overview?

File: whisper-fine-tuning-sprint/README.md [hunk 19]
```diff
@@ -0,0 +1,635 @@
...
+The section [Data and preprocessing](#data-and-preprocessing) explains 
+in more detail what audio data can be used, how to find suitable audio data, and 
+how the audio data can be processed.
+
+For training, it is recommended to use the [official training script](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py) or a modification thereof. A step-by-step guide on how to fine-tune 
+an acoustic model for a speech recognition system can be found under [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model).
+If possible it is encouraged to fine-tune the acoustic models on local GPU machines, but 
+if those are not available, the CLOUD_PROVIDER could team kindly provides a limited 
+number of GPUs for the event. Simply fill out [this google form](https://forms.gle/F2bpouvhDpKKisM39) to get access to a GPU.
+[comment]: # For more information on how to train an acoustic model on one of OVH's GPU - see [How to fine-tune a speech recognition model with OVHcould](#how-to-fine-tune-with-ovh-cloud).
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:53]: Please remove references to OVH

File: whisper-fine-tuning-sprint/README.md [hunk 20]
```diff
@@ -0,0 +1,635 @@
...
+
+During the event, the speech recognition system will be evaluated on both the Common Voice `"test"` split 
+of the participants' chosen language as well as the *real-world* `"dev"` data provided by 
+the Hugging Face team. 
+At the end of the whisper fine-tuning sprint, the speech recognition system will also be evaluated on the
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:58]: Please consistently capitalize Whisper :)

File: whisper-fine-tuning-sprint/README.md [hunk 21]
```diff
@@ -0,0 +1,635 @@
...
+*real-world* `"test"` data provided by the Hugging Face team. Each participant should add an 
+`eval.py` script to her/his model repository in a specific format that lets one easily
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:60]:
```suggestion
`eval.py` script to their model repository in a specific format that lets one easily 
```

File: whisper-fine-tuning-sprint/README.md [hunk 22]
```diff
@@ -0,0 +1,635 @@
...
+you've joined [this discord channel](https://hf.co/join/discord)
+
+Also, please make sure that you have been added to the [Speech Event Organization](https://CREATE_AN_ORGANISATION_ON_HUB). 
+You should have received an invite by email. If you didn't receive an invite, please contact the organizers, *e.g.* Sanchit or VB directly on discord.
+
+[comment]: # VB: create an organisation on the hub.
+
+## Important dates
+
+![timeline](https://github.com/patrickvonplaten/scientific_images/raw/master/Robush%20Speech%20Challenge.png)
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:85]: To update

File: whisper-fine-tuning-sprint/README.md [hunk 23]
```diff
@@ -0,0 +1,635 @@
...
+In this section, we will quickly go over how to find suitable training data and 
+how to preprocess it.
+
+To begin with, **all data except Common Voice's `"test"` data can be used as training data.**
+The exception includes all Common Voice versions as the test data split of later Common Voice versions often
+overlaps with the one of previous versions, *e.g.* the test data of Common Voice 7 in English is 
+to a big part identical to the test data of Common Voice 6 in English:
+
+```python
+load_dataset("mozilla-foundation/common_voice_11_0", "en", split="test")
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:100]: I think doing this requires previously going to the Hub and accepting the gate, so we should document that

File: whisper-fine-tuning-sprint/README.md [hunk 24]
```diff
@@ -0,0 +1,635 @@
...
+
+```python
+load_dataset("mozilla-foundation/common_voice_10_0", "en", split="test") 
+```
+
+However, we strongly encourage participants to make use of Common Voice's other splits, *e.g.* `"train"` and `"validation"`.
+For most languages, the Common Voice dataset offers already a decent amount of training data. It is usually 
+always advantageous to collect additional data. To do so, the participants are in a first step encouraged to search the
+Hugging Face Hub for additional audio data, for example by selecting the category 
+["speech-processing"](https://huggingface.co/datasets?task_categories=task_categories:speech-processing&sort=downloads).
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:113]: https://huggingface.co/datasets?task_categories=task_categories:automatic-speech-recognition is better :) 

```suggestion
["automatic-speech-recognition"](https://huggingface.co/datasets?task_categories=task_categories:automatic-speech-recognition).
```

File: whisper-fine-tuning-sprint/README.md [hunk 25]
```diff
@@ -0,0 +1,635 @@
...
+```
+
+If the above command doesn't print ``True``, in the first step, please follow the
+instructions [here](https://pytorch.org/) to install PyTorch with CUDA.
+
+We strongly recommend making use of the provided PyTorch examples scripts in [transformers/examples/pytorch/speech-recognition](https://github.com/huggingface/transformers/tree/main/examples/pytorch/speech-recognition) to train your speech recognition
+system.
+In all likelihood, you will adjust one of the example scripts, so we recommend forking and cloning the 🤗 Transformers repository as follows. 
+
+1. Fork the [repository](https://github.com/huggingface/transformers) by
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:195]: @sanchit-gandhi @Vaibhavs10 do you really think people will be doing this?

File: whisper-fine-tuning-sprint/README.md [hunk 26]
```diff
@@ -0,0 +1,635 @@
...
+inputs = feature_extractor(ds[0]["audio"]["array"], return_tensors="pt")
+input_features = inputs.input_features
+
+decoder_input_ids = torch.tensor([[1, 1]]) * model.config.decoder_start_token_id
+last_hidden_state = model(input_features, decoder_input_ids=decoder_input_ids).last_hidden_state
+```
+
+## How to finetune a whisper model
+
+In this section, we show you how to fine-tune a pre-trained [XLS-R Model](https://huggingface.co/docs/transformers/model_doc/xls_r) on the [Common Voice 7 dataset](https://huggingface.co/datasets/mozilla-foundation/common_voice_7_0).
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:267]: To update to Whisper and CV 11?

File: whisper-fine-tuning-sprint/README.md [hunk 27]
```diff
@@ -0,0 +1,635 @@
...
+
+decoder_input_ids = torch.tensor([[1, 1]]) * model.config.decoder_start_token_id
+last_hidden_state = model(input_features, decoder_input_ids=decoder_input_ids).last_hidden_state
+```
+
+## How to finetune a whisper model
+
+In this section, we show you how to fine-tune a pre-trained [XLS-R Model](https://huggingface.co/docs/transformers/model_doc/xls_r) on the [Common Voice 7 dataset](https://huggingface.co/datasets/mozilla-foundation/common_voice_7_0). 
+
+We recommend fine-tuning one of the following pre-trained XLS-R checkpoints:
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:269]: To update

File: whisper-fine-tuning-sprint/README.md [hunk 28]
```diff
@@ -0,0 +1,635 @@
...
+- [1B parameters version](https://huggingface.co/facebook/wav2vec2-xls-r-1b)
+- [2B parameters version](https://huggingface.co/facebook/wav2vec2-xls-r-2b)
+
+To begin with, please note that to use the Common Voice dataset, you 
+have to accept that **your email address** and **username** are shared with the 
+mozilla-foundation. To get access to the dataset please click on "*Access repository*" [here](https://huggingface.co/datasets/mozilla-foundation/common_voice_7_0).
+
+Next, we recommended that you get familiar with the XLS-R model and its capabilities.
+In collaboration with [Fairseq's Wav2Vec2 team](https://github.com/pytorch/fairseq/tree/main/examples/wav2vec), 
+we've written ["Fine-tuning XLS-R for Multi-Lingual ASR with 🤗 Transformers"](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2) which gives an in-detail explanation of how XLS-R functions and how it can be fine-tuned.
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:281]: To update

File: whisper-fine-tuning-sprint/README.md [hunk 29]
```diff
@@ -0,0 +1,635 @@
...
+
+The blog can also be opened and directly fine-tuned in a google colab notebook.
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:283]:
```suggestion
The blog can also be opened and directly fine-tuned in a Google Colab notebook.
```

File: whisper-fine-tuning-sprint/README.md [hunk 30]
```diff
@@ -0,0 +1,635 @@
...
+In this section, we will explain how to fine-tune the model on a local machine.
+
+1. **Log in**
+
+To begin with, you should check that you are correctly logged in and that you have `git-lfs` installed so that your fine-tuned model can automatically be uploaded.
+
+Run:
+
+```bash
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:292]: This might required much earlier before accessing the datasets

File: whisper-fine-tuning-sprint/README.md [hunk 31]
```diff
@@ -0,0 +1,635 @@
...
+huggingface-cli login
+```
+
+to login. It is recommended to login with your access token that can be found under your hugging face profile (icon in the top right corner on [hf.co](http://hf.co/), then Settings -> Access Tokens -> User Access Tokens -> New Token (if haven't generated one already)
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:296]: Update to link hf.co/settings directly

Comment (osanseviero) [reply whisper-fine-tuning-sprint/README.md:296]: Actually this is already explained now when you do `login` so this is redundant

File: whisper-fine-tuning-sprint/README.md [hunk 32]
```diff
@@ -0,0 +1,635 @@
...
+
+```bash
+git-lfs -v
+```
+
+The output should show something like `git-lfs/2.13.2 (GitHub; linux amd64; go 1.15.4)`. If your console states that the `git-lfs` command was not found, please make
+sure to install it [here](https://git-lfs.github.com/) or simply via: 
+
+```bash
+sudo apt-get install git-lfs
```
Comment (osanseviero) [inline whisper-fine-tuning-sprint/README.md:312]: FYI colab now comes with git-lfs
